sema/discovery/discovery.py

```
@@ -109,6 +110,19 @@
     def triples_found(self):
         return self._result.success
 
+    def _fetch_with_playwright(self, url: str) -> str:
```

Ruff

Missing type annotation for `self` in method

(ANN101)